client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +49,61 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    private final boolean subscriberForRetention;
+    private final boolean autoPublishAtLastCheckpoint;
+
+    public enum ReaderGroupRetentionConfig {
+        /**
+         * There is no retention done by this {@link ReaderGroup}.
+         */
+        NO_RETENTION,
```

Comment:
"NO_RETENTION" seems to indicate we don't retain any data... which is just the reverse of what this does. Could we call this "NO_CONSUMPTION_BASED_TRUNCATION" ?

Reply:
Name updated.